pkg/server/cloud/functional/functional_test_template.go

```diff
@@ -169,12 +169,18 @@ func deleteInstances(t *testing.T, c cloud.CloudClient) {
 }
 
 func ContainerAuthTest(t *testing.T, c cloud.CloudClient) {
-	username1, password1, err := c.GetRegistryAuth()
+	username1, password1, err := c.GetRegistryAuth("689494258501.dkr.ecr.us-east-1.amazonaws.com/helloserver:latest")
```

Comment:
Is this test expected to run on GCP? Thanks.

Reply:
Good point, while this is part of the CloudClient interface, this test is very AWS specific.  It should be moved into the `aws` package.  We currently don't have support for automatically creating secrets in GCP since it appears to be more complex than just generating a token.